### DISCUSSION TITLE
Vite With Prisma

### DISCUSSION THREAD

**Discussion by zwhitchcox** [remix-run/remix#8477] [Q&A]
I'm trying to migrate to vite, but I'm using prisma, and I keep getting the error `|- ReferenceError: exports is not defined` when trying to import the client. I was wondering if anyone had a suggestion to get around this?

I know [prisma does not output esm](https://github.com/prisma/prisma/issues/5030), and I haven't been able to get any modules to work with vite unless I compile them with `module: ESNext`, so I'm not sure where to go from here or if I should wait for better documentation/compatibility.

**Comment by meetdave3** [comment]
Refer the marked answer here: https://github.com/prisma/prisma/discussions/20200
